lib/_http_outgoing.js

```diff
@@ -120,6 +123,17 @@ function OutgoingMessage() {
 }
 util.inherits(OutgoingMessage, Stream);
 
+Object.defineProperty(OutgoingMessage.prototype, '_writableState', {
+  get() {
+    var state = this[kWritableState];
+    if (!state) {
+      state = new Writable.WritableState();
```

Comment:
That might be a source of bad memory leaks if someone mingles with `_writableState`. 

Another option might be to have a small class upon which we "fake" `WritableState` inheritance.